tests/test_torchinductor.py

```diff
@@ -1102,6 +1102,36 @@ def fn(x, y):
         self.assertEqual(a.stride(), c.stride())
         self.assertEqual(c.stride()[2], 1)
 
+    def test_triton_mm1(self):
+        config.triton.use_mm = True
+
+        @torchdynamo.optimize("inductor", nopython=True)
+        def fn(x, y):
+            return torch.mm(x, y)
+
+        N = 1024
+        a = torch.randn([N, N], device="cuda", dtype=torch.float16)
+        b = torch.randn([N, N], device="cuda", dtype=torch.float16)
```

Comment:
Test failures might be because you are setting device to be always "cuda". I think other tests use `self.device`.